platform-includes/getting-started-verify/javascript.nuxt.mdx

```diff
@@ -1,4 +1,35 @@
-On the client-side:
+### Server-side
+
+Sentry can only be loaded on the server-side by being explicitly added via `--import`.
+Check out the <PlatformLink to="/install/cli-import">`--import` CLI flag</PlatformLink> docs for setup instructions.
+
+To verify that Sentry works on the server-side, add the following snippet on the server-side:
+
+```js {tabTitle:Nitro} {filename:server/sentry-example-api.ts}
+import { defineEventHandler } from '#imports';
+
+export default defineEventHandler(event => {
+  throw new Error("Sentry Example API Route Error");
+});
+
+```
+
+<Alert level="warning">
```

Comment:
Not 100% sure, but do we need to make this an alert? What about instead, just making this regular text? This is part of the server-side verify step basically, isn't it?